content/docs/command-reference/params/diff.md

```diff
@@ -1,7 +1,7 @@
 # params diff
 
-Show changes in [parameter dependencies](/doc/command-reference/params) between
-commits in the <abbr>DVC repository</abbr>, or between a commit and the
+Show changes in [parameter dependencies](/doc/user-guide/concepts/parameters)
```

Comment:
We should use tooltips instead of links to `/doc/user-guide/concepts/parameters` (in most places, depends on the context) now.

Reply:
OK, good idea. Updated these ones (even though I know we'll be splitting this up).

Reply:
> Updated these ones (even though I know we'll be splitting this up

Any progress here will be preserved when I can finally split this so thanks.